frontend/packages/integration-tests-cypress/tests/i18n/pseudolocalization.spec.ts

```diff
@@ -14,38 +20,173 @@ describe('Localization', () => {
     cy.logout();
   });
 
-  it('pseudolocalizes masthead', () => {
-    cy.log('test masthead');
-    cy.visit('/dashboards?pseudolocalization=true&lng=en');
-    masthead.clickMastheadLink('help-dropdown-toggle');
-    cy.byTestID('application-launcher-item').isPseudoLocalized();
-  });
+  const tableColumnHeaders =
+    'th > .pf-c-table__button > .pf-c-table__button-content > .pf-c-table__text';
+  const sectionHeadings = '[data-test-section-heading]';
+  const detailLabels = 'dt';
+  const i18nTranslated = '[data-i18n-translated]';
 
-  it('pseudolocalizes navigation', () => {
-    cy.log('test navigation');
-    cy.visit('/dashboards?pseudolocalization=true&lng=en');
-    cy.byTestID('nav').isPseudoLocalized();
-  });
+  const createExampleResource = () => {
+    let initialUrl;
+    cy.url().then((url) => {
+      initialUrl = url;
+    });
+    listPage.clickCreateYAMLbutton();
+    cy.byTestID('resource-sidebar').should('exist');
+    yamlEditor.isLoaded();
+    yamlEditor.clickSaveCreateButton();
+    cy.get(errorMessage).should('not.exist');
+    cy.then(() => cy.visit(initialUrl));
+  };
 
-  it('pseudolocalizes activity card', () => {
-    cy.log('test activity card components');
-    cy.visit('/dashboards?pseudolocalization=true&lng=en');
-    cy.byTestID('activity').isPseudoLocalized();
-    cy.byTestID('activity-recent-title').isPseudoLocalized();
-    cy.byTestID('ongoing-title').isPseudoLocalized();
-    cy.byTestID('events-view-all-link').isPseudoLocalized();
-    cy.byTestID('events-pause-button').isPseudoLocalized();
-  });
+  const deleteExampleResource = (kind: string) => {
+    cy.url().then((url) => {
+      if (url.includes('example')) {
```

Comment:
Let's not do URL sniffing. Why not just pass the name and kind to this utility and run an oc command to delete it?